tests/e2e-leg-5/helm-nameoverride/15-assert.yaml

```diff
@@ -15,7 +15,7 @@ apiVersion: v1
 kind: Pod
 metadata:
   labels:
-    control-plane: controller-manager
+    control-plane: matt
```

Comment:
Is this on purpose?

Reply:
Actulally it is. With the change of the control-plane label, it can now be templated in the helm chart. So, in this testcase, which sets the `nameOverride` parameter, the label will be set to the value of that parameter.